es-design-system/pages/molecules/es-support.vue

```diff
@@ -1,17 +1,39 @@
 <template>
     <div>
+        <h1>
+            Default Copy
+        </h1>
         <es-support
             link="https://www.energysage.com"
-            rep-name="Ben" />
+            :src="benr" />
+        <h1 class="mt-5">
+            Custom Copy
+        </h1>
+        <es-support
+            link="https://www.energysage.com"
+            :src="benr">
+            <template #title>
+                Custom title
+            </template>
+            <template #linkCopy>
+                Link Copy
+            </template>
+        </es-support>
     </div>
 </template>
 <script>
 import { EsSupport } from '@energysage/es-vue-base';
+import benr from '@/assets/imgs/benr.png';
```

Comment:
I think we should consider using something like https://place-hold.it/300x300 instead of having a png in the project.

Reply:
Unfortunately not, the image is expected to be a certain ratio and have some transparent area around the person. I understood your argument around not including it in the vue package but whats the aversion to having it in the design system?